client/cli/src/arg_enums.rs

```diff
@@ -122,6 +122,20 @@ impl ExecutionStrategy {
 	}
 }
 
+arg_enum! {
+	/// Database backend
+	#[allow(missing_docs)]
+	#[derive(Debug, Clone, Copy)]
+	pub enum Database {
+		// Facebooks RocksDB
+		RocksDb,
+		// Subdb
+		SubDb,
+		// Subdb
```

Comment:
Is this a typo or subdb and parity db are synonms?

Reply:
Typo